q2_diversity/_beta/_method.py

```diff
@@ -63,6 +69,27 @@ def beta_phylogenetic(table: biom.Table, phylogeny: skbio.TreeNode,
     return results
 
 
+def beta_phylogenetic_hpc(table: BIOMV210Format, phylogeny: NewickFormat,
+                          metric: str, n_jobs: int=1,
+                          variance_adjusted: bool=False,
+                          alpha=1.0,
```

Comment:
I don't have a strong recommendation, but it might be nice to error if `alpha` is passed when the metric isn't generalized UniFrac. That would require you set the default to something like `None` though, which definitely obfuscates things a bit.

Reply:
I followed your suggestion here.